metabase/metabase
# Fix - Search results has a box with scrollbars

Closes #28788

### Description

I found an additional case to the original report - the overflow also occurred if a question belonged to a collection with a long name. This is included in test data.
I reported #32140 for markdown separately.

### How to verify

1. Create a collection with a very long name, ideally with also a very long word that's wider than the dropdown.
2. Create a question with very long name & description, as above.
3. Add this question to that collection.
4. Search for this question

The search dropdown should not overflow horizontally.
Horizontal scrollbar should not be shown in it.

### Before

![image](https://github.com/metabase/metabase/assets/6830683/d2b9eb2a-e804-41c1-8b23-319e7612d6ce)


### After

![image](https://github.com/metabase/metabase/assets/6830683/2c231c79-ef7b-42a4-8d41-8ce9a7b5b52b)
